src/make_parts.zsh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env zsh
```

Comment:
Aye, this was just for my local use to generate the first draft files.  The script and files were outdated by the factory function.